client/go/dogma/content_service.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2018 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package dogma
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type contentService service
+
+// Query specifies a query on a file.
+type Query struct {
+	Path string
+	// QueryType can be "identity" or "json_path". "identity" is used to retrieve the content as it is.
+	// "json_path" applies a series of JSON path to the content.
+	// See https://github.com/json-path/JsonPath/blob/master/README.md
+	QueryType   string
```

Comment:
sorry I missed that at first round, do you think string is better than iota? 
Using iota may need some more boilerplate code to generate equivalent string, but for users it's easier to use because they know that what type of QueryType is supported, and prevent typo as well.

Reply:
`iota` should be much better~! Thank you~!